packages/block-library/src/cover/style.scss

```diff
@@ -9,7 +9,6 @@
 	display: flex;
 	justify-content: center;
 	align-items: center;
-	overflow: hidden;
```

Comment:
This was added originally to support "cropping" video backgrounds but in my testing, it's not needed anymore.